camera-manager.js

```diff
@@ -206,6 +217,25 @@ class CameraManager extends EventTarget {
     }
     return result;
   }
+  
+  //Lerp towards offset for smoother ADS, works both aiming in and out
+  ads(){
+    if (localPlayer.hasAction('aim') && -cameraOffset.z>1){
```

Comment:
`-cameraOffset.z>1` should probably not be hardcoded, could add it as a variable above so it's easier to edit.

Reply:
This was an oversight while I was testing different distances, but it now uses the inverted value from "maxAim" an already existing variable above